components/modal/Register/register.module.scss

```diff
@@ -0,0 +1,50 @@
+.errorPrompt {
+  color: red;
+}
+
+.select {
+  width: 300px;
+}
+.tagWord {
+  margin: 10px;
+  margin-top: 10px;
+  font-family: 'Mono', sans-serif;
+  font-size: 1rem;
+  font-weight: 500;
+}
+
+.inputBox {
+  padding: 10px;
+  border: 1px solid #1d1283;
+  width: 100%;
+  border-radius: 5px;
+  font-size: 15px;
+  margin-top: 5px;
+}
+
+.inputBoxCur {
+  padding: 10px;
+  border: 1px solid #1d1283;
+  width: 70%;
+  border-radius: 5px;
+  font-size: 15px;
+}
+
+.submitButton {
+  font-size: 1.2rem;
+  padding: 8px 15px;
+  background-color: rgb(238, 26, 117);
+  color: #fff;
+  border-radius: 8px;
+  transition-duration: 0.1s;
+  cursor: pointer;
+  display: block;
+  margin: 0 auto;
+  margin-top: 10px;
+  border: rgb(238, 26, 117);
+}
+.submitButton:hover {
+  zoom: 110%;
+  background-color: #1d1283;
+  color: #fff;
+}
```

Comment:
can we use `rem` wherever possible? I can see some places you are using `px` and some places `rem` 
let's stick with one pattern, `rem`